### PR TITLE
Add `last_hit` to uft hit/expire probes, prevent `Moment` time delta wrapping

### PR DESCRIPTION
**Pull request by FelixMcFelix**
Adds some extra probes to help us track down and verify #426, and also
slightly changes up `Moment::delta_as_millis()` to saturate if the order
if `now` and `earlier` are seemingly back to front (as is apparently
now standard behaviour in std).

### CODE REVIEW

File: dtrace/opte-flow-expire.d [hunk 1]
```diff
@@ -8,20 +8,22 @@
...
+	printf(HDR_FMT, "PORT", "FT NAME", "FLOW", "LAST_HIT", "NOW");
 	num = 0;
 }
 
 flow-expired {
 	this->port = stringof(arg0);
 	this->name = stringof(arg1);
 	this->flow = (flow_id_sdt_arg_t *)arg2;
+	this->last_hit = stringof(arg3);
+	this->now = stringof(arg4);
```
Comment (citrus-it) [inline dtrace/opte-flow-expire.d:23]: Are these really strings, and not u64s?

File: lib/opte/src/ddi/time.rs [hunk 2]
```diff
@@ -75,6 +77,19 @@ impl Moment {
             }
         }
     }
+
+    /// Return the underlying timestamp for debugging purposes
+    /// if supported on the current platform.
+    #[allow(dead_code)]
```
Comment (rcgoodfellow) [inline lib/opte/src/ddi/time.rs:83]: I presume this is because we're only calling this in a `no-std` context?

**Review by rcgoodfellow — APPROVED** [review]
LGTM. Just one non-blocking question.

File: lib/opte/src/ddi/time.rs (hunk 2 above)
Comment (FelixMcFelix) [reply lib/opte/src/ddi/time.rs:83]: It is, yes.

File: dtrace/opte-flow-expire.d (hunk 1 above)
Comment (FelixMcFelix) [reply dtrace/opte-flow-expire.d:23]: Good catch, I'll convert this to match opte-uft-hit's stucture.

**Pull request merged**